### PR TITLE
Publish August 17 with 14286: update browser agent size stats

### PR DESCRIPTION
**Pull request by patrickhousley**
Please follow [conventional commit standards](https://www.conventionalcommits.org/en/v1.0.0/)
in your commit messages and pull request title.

## Give us some context

Updating the browser agent size stats for the upcoming 1.238.0 release.

### CODE REVIEW

**Comment by patrickhousley** [comment]
@cwli24 @metal-messiah Can you review this please?
Docs team, please wait to merge until we get 1.238.0 release PR up.

For the record, the values that were replaced here were outdated. Just so we can see that this is an overall improvement, below are the stats from 1.237.1:

|loader|inline script|async chunks|
|---|---|---|
|lite|10.5|25.5
|pro|15.8|39.5|
|spa|17.8|46.85|

All the stats were derived using our local test server with gzip compression and loading the assets via chrome latest.

**Comment by bradleycamacho** [comment]
I'll wait for an approval from the folks you tagged before merging this. Feel free to ping me on slack to get this merged in ASAP!

**Comment by cwli24** [comment]
@bradleycamacho we conversed, I'm good 👍 thanks

**Review by homelessbirds — APPROVED** [review]

**Pull request merged**
